src/bindgen/config_reader/condition_evaluator.cr

```diff
@@ -25,7 +29,7 @@ module Bindgen
       # Regular expression for conditionals.
       # Matches: `[els]if VARIABLE (is|matches) VALUE`.
       # Instead of a space, an underscore may be used instead.
-      RX = /^(?:els)?if(?: +|_)(.+?)(?: +|_)(is|isnt|matches)(?: +|_)(.*)$/
+      RX = /^(?:els)?if(?: +|_)(.+?)(?: +|_)(is|isnt|matches|newer|older|newer_or_equals|older_or_equals)(?: +|_)(.*)$/
```

Comment:
This introduces `newer_or_equals` and `older_or_equals` that are not handled nor documented - Oversight?